src/posix/platform/openthread-system.h

```diff
@@ -90,12 +90,17 @@ enum
  */
 typedef struct otPlatformConfig
 {
-    uint64_t    mNodeId;        /// Unique node ID.
-    const char *mInterfaceName; /// Thread network interface name.
-    const char *mRadioFile;     /// Radio file path.
-    const char *mRadioConfig;   /// Radio configurations.
-    uint32_t    mSpeedUpFactor; /// Speed up factor.
-    bool        mResetRadio;    /// Whether to reset RCP when initializing.
+    uint32_t    mSpeedUpFactor;     ///< Speed up factor.
+    uint8_t     mLogLevel;          ///< Debug level of logging.
+    bool        mIsDryRun;          ///< Dry run.
+    bool        mPrintRadioVersion; ///< Whether to print radio firmware version.
```

Comment:
Why put `IsDryRun` and `PrintRadioVersion` here? They are really temporary necessary.

Reply:
Just want to put all parsed parameters in s single structure. I have updated the code to use another method to process this problem.